package.json

```diff
@@ -90,7 +92,7 @@
     "react-dom": "^18",
     "react-ga": "^3.3.1",
     "react-gtm-module": "^2.0.11",
-    "react-i18next": "^14.0.5",
+    "react-i18next": "^14.1.3",
```

Comment:
Do we still need this library?

Reply:
@sambhavgupta0705 We should uninstall the react-i18next library, i don't we need it anymore.

Reply:
Yes I will fix this one